Firmware/ultralcd.cpp

```diff
@@ -3843,26 +3843,18 @@ static void lcd_show_sensors_state()
     //  Shutter register is an index of LASER shutter time. It is automatically controlled by the chip's internal
     //  auto-exposure algorithm. When the chip is tracking on a good reflection surface, the Shutter is small.
     //  When the chip is tracking on a poor reflection surface, the Shutter is large. Value ranges from 0 to 46.
-	if (mmu_enabled == false)
-	{
-		//if (!fsensor_enabled)
-		//	lcd_puts_P(_N("Filament sensor\n" "is disabled."));
-		//else
-		//{
-		if (!moves_planned() && !IS_SD_PRINTING && !is_usb_printing && (lcd_commands_type != LcdCommands::Layer1Cal))
-			pat9125_update();
-			lcd_set_cursor(0, 2);
-			lcd_printf_P(_N(
-				"Xd:  %3d  "
-				"Yd:  %3d\n" ////c=4
-				"Int: %3d  " ////c=4
-				"Shut:  %3d"  ////c=4
-			),
-				pat9125_x, pat9125_y,
-				pat9125_b, pat9125_s
-			);
-		//}
-	}
+    if (mmu_enabled == false)
+    {
+        // pat9125_update is already called while printing: only update manually when idling
+        if (!moves_planned() && !IS_SD_PRINTING && !is_usb_printing && (lcd_commands_type != LcdCommands::Layer1Cal))
+            pat9125_update();
+
+        lcd_set_cursor(0, 2);
+        lcd_printf_P(_N(" Xd: %4d    Yd: %3d\n"
```

Comment:
What's with this change in the fmt? Wouldn't it make more sense to have it %6d as to handle the entire signed 16 bit number? I'd also change the spaces a bit if I did this.

Also please update the doxygen menu "render" if you change the layout a bit

Reply:
The formatting was badly aligned, it was looking out of place.
While printing this is unlikely to get anywhere higher than 100 as it's reset regularly.

But I was able to increase this by cheating, push filament in super-fast, then slowly pull it out (or just do the opposite), so that I could go higher.

I rearranged the layout so that we can show a full %6d. Looks a bit empty by comparison:

![2022-02-14T233407](https://user-images.githubusercontent.com/1017726/153957975-76e670f1-9a08-48bb-b737-082af66f7d8d.png)